db/migrate/20200723225140_create_campaigns.rb

```diff
@@ -0,0 +1,16 @@
+class CreateCampaigns < ActiveRecord::Migration[6.0]
+  def change
+    create_table :campaigns do |t|
+      t.boolean :active, default: false
+      t.boolean :is_valid, default: false
```

Comment:
Could we make this valid or make active is_active?

Reply:
I had an issue earlier because I think `ActiveRecord` has a `valid` method. I installed [`safe_attributes`](https://github.com/bjones/safe_attributes) and updated to `valid`.